db/migrate/20170721141401_create_taggings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTaggings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :taggings do |t|
+      t.integer :tag_id
+      t.integer :taggable_id
+      t.string :taggable_type
+
```

Comment:
@case-eee I need the same tags to be applied on multiple models, so that Dailies, Journals, and Resources can all access tags through that joins table. Since it's one joins table that could be applied to any of three different models, that's where the polymorphism comes in.